multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/steps/ValidateDeployParametersStep.java

```diff
@@ -72,6 +79,7 @@ private void validateExtensionDescriptorFileIds(ProcessContext context) {
         for (String fileId : extensionDescriptorFileIds) {
             FileEntry file = findFile(context, fileId);
             validateDescriptorSize(file);
+            extensionDescriptors.add(file);
```

Comment:
Can we just use same logic using local variable similar to `archivePartEntries` instead of add them into class field?